composer.json

```diff
@@ -26,13 +26,13 @@
     "require-dev": {
         "apigen/apigen": "^2.8",
         "pdepend/pdepend": "^2.2",
-        "phpmd/phpmd": "^2.0",
-        "phpunit/php-invoker": "^1.1",
-        "phpunit/phpunit": "^4.0",
         "sebastian/finder-facade": "^1.1",
         "sebastian/phpcpd": "^1.4",
+        "theseer/fdomdocument": "^1.6",
+        "phpmd/phpmd": "^2.0",
         "squizlabs/php_codesniffer": "^1.4",
-        "theseer/fdomdocument": "^1.6"
+        "fabpot/php-cs-fixer": "^0.3|^1.11"
```

Comment:
For local use global tools. or, if really, really need - then install SCA tools same way as Travis will.

> Also the build is going to be non-deterministic.

Why is that?